.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: false
+language: python
+
+cache:
+  directories:
+    - "$HOME/google-cloud-sdk/"
```

Comment:
Cool.  I didn't realize you could cache stuff.  Does it try to rerun using the same machine?